release.json

```diff
@@ -1,28 +1,28 @@
 {
     "base_branch": "main",
     "last_stable": {
-        "6": "6.35.0",
-        "7": "7.35.0"
+        "6": "6.35.1",
+        "7": "7.35.1"
     },
     "nightly": {
-        "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "master",
-        "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+        "INTEGRATIONS_CORE_VERSION": "7.36.x",
+        "OMNIBUS_SOFTWARE_VERSION": "7.36.x",
+        "OMNIBUS_RUBY_VERSION": "7.36.x",
```

Comment:
Was updating this file intentional? This looks like it was mistakenly merged from `7.36.x`

Reply:
I think the base branch is just wrong